### DISCUSSION TITLE
Running with own mail server

### DISCUSSION THREAD

**Discussion by Kuturkokov** [simple-login/app#714] [Self-hosting question] [answered]
Hello,
I have my own mail server and wanted to run these containers on a different network.
In the setup it says to remove all MX and only point to Simple Login.

How would Simple login then deliver to my main address on a different server?

**Comment by nguyenkims** [comment, marked as answer]
@Kuturkokov a domain can only point to a single email service so if your domain is already handled by your own mail server, this domain can't be used for SimpleLogin.

**Reply by Kuturkokov** [reply]
Thanks. I will set up a subdomain to handle the aliases.
